app/utils/command.py

```diff
@@ -0,0 +1,81 @@
+import inspect
+import logging
+from typing import Any
+
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandParser
+
+
+class CustomBaseCommand(BaseCommand):
+
+    def handle(self, *args: Any, **options: Any) -> None:
+        """
+        The actual logic of the command. Subclasses must implement
+        this method.
+        """
+        raise NotImplementedError("subclasses of CustomBaseCommand must provide a handle() method")
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument('--logger', action='store_true', help='use logger configuration')
+
+
+class CommandHandler():
+    """Base class for management command handler
+
+    This class add proper support for printing to the console for management command
```

Comment:
For my understanding, what happens without this? I thought that this whole BOD migration would be mostly a locally run CLI tool, so no need for more advanced logging things.

Reply:
Yes, not really needed here but  mainly for consistency with `stac` and future commands (such as synchronize with cognito).